### PR TITLE
set the json name of exitCode

### PR DESCRIPTION
**Pull request by davidstack**
set the json name of exitCode ,so we can specify the exitcode in the policies.
       policies:
              - exitCode: 137
               action: RestartJob

### CODE REVIEW

File: pkg/apis/batch/v1alpha1/job.go [hunk 1]
```diff
@@ -193,7 +193,7 @@ type LifecyclePolicy struct {
 	// according to this code.
 	// Note: only one of `Event` or `ExitCode` can be specified.
 	// +optional
-	ExitCode *int32
+	ExitCode *int32 `json:"exitCode,omitempty" protobuf:"bytes,3,opt,name=exitCode"`
```
Comment (k82cn) [inline pkg/apis/batch/v1alpha1/job.go]: Good catch!

It should be `5`, `3` is used by `Events` :)

Comment (davidstack) [reply pkg/apis/batch/v1alpha1/job.go]: got it,i have changed this.

**Pull request merged**